openedx/adg/admin_dashboard/applications/tests_applications/test_applications_helpers.py

```diff
@@ -0,0 +1,135 @@
+        ApplicationHubFactory(user=user, submission_date=date)
+
+
+@skip_unless_lms
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    'is_superuser, group_name, expected_count',
+    [
+        (True, None, 2),
+        (False, ADG_ADMIN_GROUP_NAME, 2),
+        (False, BUSINESS_LINE_TEST_NAME_1, 1)
+    ],
+    ids=['super_user', 'adg_admin', 'business_line_admin']
+)
+def test_get_submitted_applications(is_superuser, group_name, expected_count):
+    """
+    Test `get_submitted_applications` function for different types of admins.
+    """
+    user_1 = UserFactory()
+    user_2 = UserFactory()
+
+    business_line_1 = BusinessLineFactory(title=BUSINESS_LINE_TEST_NAME_1)
+    business_line_2 = BusinessLineFactory(title=BUSINESS_LINE_TEST_NAME_2)
+
+    ApplicationHubFactory(user=user_1, is_written_application_completed=True)
+    ApplicationHubFactory(user=user_2, is_written_application_completed=True)
+
+    UserApplicationFactory(user=user_1, business_line=business_line_1)
+    UserApplicationFactory(user=user_2, business_line=business_line_2)
+
+    if is_superuser:
+        admin_user = UserFactory(is_staff=True, is_superuser=True)
+    else:
+        admin_user = UserFactory(is_staff=True, groups=[GroupFactory(name=group_name)])
+
+    submitted_applications = get_submitted_applications(admin_user)
+
+    assert submitted_applications.count() == expected_count
+
+
+@skip_unless_lms
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    'submissions_previous_month, submissions_current_month, expected_new_submissions, expected_percentage',
+    [
+        (0, 0, 0, 0),
+        (1, 1, 1, 0),
+        (2, 1, 1, -50),
+        (1, 2, 2, 100)
+    ],
+    ids=['zero_submissions', 'same_submissions', 'less_submissions', 'more_submissions']
+)
+def test_get_new_applications_and_percentage_change(
+    submissions_current_month, submissions_previous_month, expected_new_submissions, expected_percentage
+):
+    """
+    Test `get_new_applications_and_percentage_change` function for different number of submissions for the current and
+    previous month.
+    """
+    today = datetime.today()
+    create_applications(submissions_previous_month, datetime(today.year, today.month - 1, 1))
+    create_applications(submissions_current_month, today)
+
+    new_submissions, percentage = get_new_applications_and_percentage_change(UserApplication.objects.all())
+
+    assert new_submissions == expected_new_submissions
+    assert percentage == expected_percentage
+
+
+@skip_unless_lms
+@pytest.mark.django_db
+def test_get_weekly_application_submissions():
+    """
+    Test `get_weekly_application_submissions` function
+    """
+    submissions_list = [1, 2, 3, 1, 2, 3, 3, 0, 1, 1, 2, 0]
+    date_list = []
+
+    today = datetime.today()
+    date = today - timedelta(days=today.weekday())
+
+    for submissions in reversed(submissions_list):
+        create_applications(submissions, date)
+        date_list.insert(0, date.strftime("%b %d"))
```

Comment:
single qoutes. We can also move the date format to constants if it is being used more than once in this PR.